tensorflow/python/ops/image_ops_impl.py

```diff
@@ -356,6 +356,24 @@ def random_flip_up_down(image, seed=None):
     A tensor of the same type and shape as `image`.
   Raises:
     ValueError: if the shape of `image` not supported.
+    
+  Usage Example:
+    ```python
+    >> x = tf.random.normal(shape=(2, 2, 3))
```

Comment:
```suggestion
    >>> x = tf.random.normal(shape=(2, 2, 3))
```

Reply:
Hello, thanks for the suggestions but I did all of them on my IDE due to some worries about linting, blank lines etc. You can see the problem is resolved with these commits:
 https://github.com/tensorflow/tensorflow/pull/35388/commits/469e0cd5b66aadc6ac352487633c1cebf9d7cf15

https://github.com/tensorflow/tensorflow/pull/35388/commits/2a64b37de0ebddca511404d88c5a28b323630cfc